docs/development/ide-configuration.md

```diff
@@ -111,8 +111,8 @@ To be able to run tests from IntelliJ, you must set up the default Jest template
 
 Adjust following parameters:
```

Comment:
We need to remove the configuration file in the template Jest, so that IntelliJ can configure Jest with the correct configuration automatically.

Reply:
yes true story, I have added documentation for that